packages/hoppscotch-common/src/components/history/index.vue

```diff
@@ -3,7 +3,7 @@
     <div
       class="sticky top-0 z-10 flex flex-shrink-0 flex-col overflow-x-auto border-b border-dividerLight bg-primary"
     >
-      <WorkspaceCurrent :section="t('tab.history')" />
+      <WorkspaceCurrent :section="t('tab.history')" :is-only-personal="true" />
```

Comment:
Is this intentional? Is history not synced to the team workspace currently?

Reply:
History entries are personal hence won't be synced across the team. Somehow missing this notation went unnoticed all this time.